config/routes.rb

```diff
@@ -126,6 +126,7 @@
   get "/docs/agent/upgrading",           to: redirect("/docs/agent/v3/upgrading",                  status: 301)
   get "/docs/agent/upgrading-to-v3",     to: redirect("/docs/agent/v3/upgrading",                  status: 301)
   get "/docs/clusters/queue-metrics", to: redirect("/docs/pipelines/cluster-queue-metrics",        status: 301)
+  get "/docs/test-analytics/java", to: redirect("/docs/test-analytics/importing-junit-xml",        status: 301)
```

Comment:
not a biggie but it would be nice to align the `to:` on this line and the line previous!